JesusFilm/core
# fix: slow delete block

# Description

### Issue

The delete step dialog and quick control menu doesn't disable on database call.
[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37071143/todos/7409941009)

### Solution

Add loading to the dialog delete and quick control menu

# External Changes

n/a

# Additional information

n/a

# How to test
Delete a step block in the journey flow. When presented with the are you sure dialog, try and mash the delete button multiple times. You should not be able to because the button should disable.

Delete a block in the canvas editor. When presented with the quick controls menu, try and mash the delete button multiple times. You should not be able to because the button should disable.
